plugin/src/main/java/org/wildfly/plugins/bootablejar/maven/goals/AbstractDevBootableJarMojo.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wildfly.plugins.bootablejar.maven.goals;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.wildfly.core.launcher.BootableJarCommandBuilder;
+import org.wildfly.plugins.bootablejar.maven.common.Utils;
+
+/**
+ * Build and start a bootable JAR for dev-watch mode. This goal monitors the
+ * changes in the project and recompile/re-deploy. Type Ctrl-C to kill the
+ * running server.
+ *
+ * @author jfdenise
+ */
+public abstract class AbstractDevBootableJarMojo extends BuildBootableJarMojo {
+    /**
+     * Additional JVM options.
+     */
+    @Parameter(property = "wildfly.bootable.jvmArguments")
+    public List<String> jvmArguments = new ArrayList<>();
+
+    /**
+     * Bootable JAR server arguments.
+     */
+    @Parameter(property = "wildfly.bootable.arguments")
+    public List<String> arguments = new ArrayList<>();
+
+    @Override
+    public void execute() throws MojoExecutionException, MojoFailureException {
+        if (skip) {
+            getLog().debug(String.format("Skipping run of %s:%s", project.getGroupId(), project.getArtifactId()));
+            return;
+        }
+        configureServer();
+        hollowJar = true;
+        super.execute();
+        doExecute();
+    }
+
+    protected abstract void configureServer();
+
+    protected abstract void doExecute() throws MojoExecutionException, MojoFailureException;
+
+    protected BootableJarCommandBuilder buildCommandBuilder() throws MojoExecutionException {
+        return BootableJarCommandBuilder.of(Utils.getBootableJarPath(null, project, "dev"))
+                // Always disable color when printing to file.
+                .addJavaOption("-Dorg.jboss.logmanager.nocolor=true")
```

Comment:
If it's required that the color output is disabled we'll need to set this last. If it's not required this is fine.

Reply:
Actually I am wandering why we set it in all cases. We now have the ability to inherit or redirect to a file. We should only set it when we redirect to a file. I checked and the problem is already in the master branch.
Seems that we need to add it last only if we redirect to a file.

Reply:
That would make sense. We probably added it because of the redirect.

Reply:
Done.